### ISSUE TITLE
Needs Updated

### ISSUE THREAD

**Issue by TallJeff** [MicrosoftDocs/vsts-docs#1031]
This would be a great source of information if it were updated to include TFS 2018 Update 1 and TFS 2018 Update 2

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ff0c895d-9660-6db7-676e-5aa194b403e8
* Version Independent ID: 5c14f6ec-f4b5-26dc-aaf2-d8fc445036b5
* Content: [What's new for Team Foundation Server](https://docs.microsoft.com/en-us/tfs/server/whats-new)
* Content Source: [docs/server/whats-new.md](https://github.com/MicrosoftDocs/vsts-docs/blob/live/docs/server/whats-new.md)
* Product: **devops-server**
* GitHub Login: @elbatk
* Microsoft Alias: **elbatk**

**Comment by steved0x** [comment]
@TallJeff We'll get this updated, in the meantime you can get some of the information about what's new from here: [VSTS Features timeline - current features](https://docs.microsoft.com/en-us/vsts/release-notes/#current-features) - even though it is called "VSTS Features timeline" it includes the TFS versions as well.

**Comment by elbatk** [comment]
Yes, sorry @TallJeff I've been out of office but I'll get this updated!

**Comment by TallJeff** [comment]
Thanks!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
